parser/prism/Translator.cc

```diff
@@ -551,7 +580,7 @@ unique_ptr<parser::Node> Translator::translate(pm_node_t *node) {
             unreachable("PM_ENSURE_NODE is handled separately as part of PM_BEGIN_NODE, see its docs for details.");
         }
         case PM_FALSE_NODE: { // The `false` keyword
-            return make_unique<parser::False>(location);
+            return make_node_with_expr<parser::False>(MK::False(location), location);
```

Comment:
Are these desugared in this PR because they are very simple?

Reply:
`PrismLhsNode` desugaring is a bit more complicated. If they aren't needed we could simplify the PR by omitting them for later.